docs/UPGRADE_V1.6.0_TO_V1.7.0.md

```diff
@@ -0,0 +1,274 @@
+# Upgared Hub v1.6.0 to v1.7.0
+
+This doc defines the steps to follow for upgrading the Hub v1.6.0 to Hub v1.7.0.
+
+Navigate to `config` directoy in the root of the project and considering the `tekton-hub` namsespace.
+
+## Database Deploy
```

Comment:
It would be nice to get the backup of database first and then create a new deployment of db